plugins/presentation/src/components/internal/Table.svelte

```diff
@@ -114,15 +113,15 @@
 <div class="erp-table">
   <div class="thead">
     <div class="tr">
-      {#each attributes as attr (attr.key)}
+      {#each attributes as attr ({ _class: attr._class, key: attr.key })}
         <div class="th">{attr.label}</div>
       {/each}
     </div>
   </div>
   <div class="tbody">
     {#each objects as object (object._id)}
       <div class="tr" on:click={() => dispatch('open', { _id: object._id, _class: _class })}>
-        {#each attributes as attr (attr.key)}
+        {#each attributes as attr ({ _class: attr._class, key: attr.key })}
```

Comment:
Do such keys work properly? I'm just not sure how svelte compares keys so in case it uses refs probably we have a problem here